src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/attachment-unit-form/attachment-unit-form.component.ts

```diff
@@ -138,4 +149,14 @@ export class AttachmentUnitFormComponent implements OnInit, OnChanges {
             this.fileName = formData?.fileProperties?.fileName;
         }
     }
+
+    /**
+     * Set the translation strings
```

Comment:
Why the same comment as the name of the method?

Reply:
I agree. This comment is currently redundant. 
You should explain what exactly the "translation strings" are.

Also it might make more sense to rename this method to "updateTranslationStrings", but I'll leave this up to you as it is more a personal preference from my side.

Reply:
rename the comment